fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/filter/LoanCOBApiFilter.java

```diff
@@ -29,9 +29,11 @@
 import org.apache.fineract.infrastructure.jobs.exception.LoanIdsHardLockedException;
 import org.apache.fineract.useradministration.exception.UnAuthenticatedUserException;
 import org.apache.http.HttpStatus;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.web.filter.OncePerRequestFilter;
 
 @RequiredArgsConstructor
+@ConditionalOnProperty(value = "fineract.job.loan-cob-enabled", havingValue = "true")
```

Comment:
Should we have @conditional(LoanCOBEnabledCondition.class) instead? I might be wrong but Loan COB filter can be executed only on manager, right?

Reply:
changed